pkg/command/service/measure.go

```diff
@@ -494,8 +494,13 @@ kperf service measure --svc-perfix svc --range 1,200 --namespace ns --concurrenc
 				"ingress_config_ready",
 				"ingress_lb_ready"}}, rawRows...)
 			total := measureFinalResult.readyCount + measureFinalResult.notReadyCount + measureFinalResult.notFoundCount + measureFinalResult.failCount
+			knativeVersion := getKnativeVersion(p)
 			if measureFinalResult.readyCount > 0 {
 				fmt.Printf("-------- Measurement --------\n")
+				fmt.Printf("Basic Information:\n")
+				fmt.Printf("  - Versions:\n")
+				fmt.Printf("    Serving: %v\n", knativeVersion["serving"])
+				fmt.Printf("    Eventing: %v\n", knativeVersion["eventing"])
```

Comment:
As discussed in the PR comment, please add the networking solution version, like Istio